dotnet/src/Connectors/Connectors.AI.OpenAI/Connectors.AI.OpenAI.csproj

```diff
@@ -26,5 +26,6 @@
 
   <ItemGroup>
     <PackageReference Include="Azure.AI.OpenAI" />
+    <PackageReference Include="JsonSchema.Net.Generation" />
```

Comment:
Should this be a dependency here?

Reply:
Yes the OpenAI connector needs to be able to generate the Schemas so it needs this library.